python/lsst/ci/build.py

```diff
@@ -188,6 +190,7 @@ def _build_product(self, product, progress):
             # execute the build file from the product directory, capturing the output and return code
             process = subprocess.Popen(buildscript, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=productdir)
             for line in iter(process.stdout.readline, ''):
+                line = "[%s] %s" % (datetime.datetime.utcnow().isoformat(), line)
```

Comment:
Must have a timezone indicator (at least "Z") here.  If we're keeping this for much longer, I think I'd want to consider using logging instead of custom output.